studio/src/pages/[organizationSlug]/graph/[slug]/checks/[checkId]/index.tsx

```diff
@@ -154,6 +155,8 @@ const CheckOverviewPage: NextPageWithLayout = () => {
     ? "No operations were affected by breaking changes"
     : "All tasks were successful";
 
+  const isSubgraphDeletion = sdl.match(`{ deleteSubgraph }`);
```

Comment:
This state should be properly stored in the database and not identified by a fixed schema string.

Reply:
You can create a `deleted` column in the `schema_checks` table.